clouddrift/sphere.py

```diff
@@ -5,10 +5,12 @@
 """
 
 import numpy as np
-from typing import Optional, Tuple
+from typing import Optional, Tuple, Union
 import xarray as xr
+import warnings
 
 EARTH_RADIUS_METERS = 6.3781e6
+EARTH_ROTATION_RATE = 7.2921159e-5
```

Comment:
We can get the exact value like this. With the Earth Rotation being 23 hours 56 minutes 4.091 seconds.
```suggestion
EARTH_ROTATION_RATE = 2*np.pi/86164.091
```

Reply:
```suggestion
EARTH_ROTATION_RATE = 7.2921159e-5
EARTH_DAY_SECONDS = 86164.091
EARTH_ROTATION_RATE = 2 * np.pi / EARTH_DAY_SECONDS
```